### ISSUE TITLE
What should be filled in the default of --fold in train.py

### ISSUE THREAD

**Issue by EmarkZOU** [hbliu98/CLFC-Brain-Tumor-Segmentation#1]
Congratulations on your excellent academic achievements！However，When I reproduced it to work, I ran into some problems.When I ran train.py, I encountered the following error.
![image](https://github.com/hbliu98/CLFC-Brain-Tumor-Segmentation/assets/43851022/7297f942-a81c-42ec-ab96-a288a063fdf6)
I don't know what should be filled in the default in "parser.add_argument('--fold', help='which data fold to train on', required=True, type=int)".
Looking forward to your answer, thank you

**Comment by samulenzz** [comment]
> Congratulations on your excellent academic achievements！However，When I reproduced it to work, I ran into some problems.When I ran train.py, I encountered the following error.
> ![image](https://user-images.githubusercontent.com/43851022/241858049-7297f942-a81c-42ec-ab96-a288a063fdf6.png)
> I don't know what should be filled in the default in "parser.add_argument('--fold', help='which data fold to train on', required=True, type=int)".
> Looking forward to your answer, thank you

You should try this，`python train.py --fold 0`。
`FOLD` is K-FOLD-cross-validations and it can be `0, 1, 2, ..., k-1`

**Comment by EmarkZOU** [comment]
> > Congratulations on your excellent academic achievements！However，When I reproduced it to work, I ran into some problems.When I ran train.py, I encountered the following error.
> > ![image](https://user-images.githubusercontent.com/43851022/241858049-7297f942-a81c-42ec-ab96-a288a063fdf6.png)
> > I don't know what should be filled in the default in "parser.add_argument('--fold', help='which data fold to train on', required=True, type=int)".
> > Looking forward to your answer, thank you
> 
> You should try this，`python train.py --fold 0`。 `FOLD` is K-FOLD-cross-validations and it can be `0, 1, 2, ..., k-1`

Thank you for your reply！

**Comment by yanjy5** [comment]
Congratulations on the great results of your research! However, when I reproduce your work, I ran into the following problem. Specifically, when using the IXI dataset to reconstruct normal brain images, the dataloader produces the following error:
![image](https://github.com/hbliu98/CLFC-Brain-Tumor-Segmentation/assets/109884510/a6ebb1cc-17bc-410b-b5ac-0dbc1c5c2b9a)
I don't know why dataworker didn't work, all the settings are same as the orginal code.
Looking forward for you reply!

**Comment by samulenzz** [comment]
> Congratulations on the great results of your research! However, when I reproduce your work, I ran into the following problem. Specifically, when using the IXI dataset to reconstruct normal brain images, the dataloader produces the following error:
> ![image](https://user-images.githubusercontent.com/109884510/263282535-a6ebb1cc-17bc-410b-b5ac-0dbc1c5c2b9a.png)
> I don't know why dataworker didn't work, all the settings are same as the orginal code.
> Looking forward for you reply!

I found similar error (`sampler option is mutually exclusive with shuffle` ) in stackOverFlow
https://stackoverflow.com/questions/61033726/valueerror-sampler-option-is-mutually-exclusive-with-shuffle-pytorch

Did you set `shuffle = true` in line 30? It should be **false**.
https://github.com/hbliu98/CLFC-Brain-Tumor-Segmentation/blob/cdd9e9cb7c8acfe67f3aceb21295ec13833dcefa/Synthesis/dataset/dataset.py#L30

**Comment by yanjy5** [comment]
Thank you for your advice! I have tried to set "shuffle = true", but it didn't work. However, when I tried to set "shuffle_subjects = False", dataloader can work.
![image](https://github.com/hbliu98/CLFC-Brain-Tumor-Segmentation/assets/109884510/af90eff5-5279-468c-bfbe-fece47070864)

Unfortunately, another problem occurred. After I modify the dataset file, when I run the train.py file in Synthesis, the following error occors in function.py：
![image](https://github.com/hbliu98/CLFC-Brain-Tumor-Segmentation/assets/109884510/6bd40d16-b06d-4ee1-af5a-216dd1504d2d)
